now.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": 2,
+  "public": true,
+  "scope": "aragon",
+  "github": {
+    "silent": true
+  },
+  "build": {
+    "env": {
+      "CHAIN_ID": "4",
+      "FORTMATIC_API_KEY": "@aragon-client-fortmatic-testnet-api-key",
+      "PORTIS_DAPP_ID": "@aragon-client-portis-dapp-id",
```

Comment:
We can remove Fortmatic and Portis from the Rinkeby build as well; they won't be useful in a development environment :).